packages/auth/src/SignUp/SignUpForm.tsx

```diff
@@ -58,13 +58,24 @@ export const SignUpForm: FC<SignUpFormProps> = ({ withCompanyName = true }) => {
         })}
       >
         <FForm>
-          <FInput name='name' size='large' placeholder={t('auth.sign-up.form.name')} />
-          <FInput name='email' type={'email'} size='large' placeholder={t('auth.sign-up.form.email')} />
+          <FInput name='name' size='large' placeholder={t('auth.sign-up.form.name')} data-test-id='name-box' />
+          <FInput
+            name='email'
+            type={'email'}
+            size='large'
+            placeholder={t('auth.sign-up.form.email')}
+            data-test-id='email-box'
+          />
           {withCompanyName && (
-            <FInput name='companyName' size='large' placeholder={t('auth.sign-up.form.company-name')} />
+            <FInput
+              name='companyName'
+              size='large'
+              placeholder={t('auth.sign-up.form.company-name')}
+              data-test-id='compenyName-box'
+            />
           )}
 
-          <FButton type='submit' fullWidth variant='primary' loading={loading}>
+          <FButton type='submit' fullWidth variant='primary' loading={loading} data-test-id='signupSubmit-btn'>
```

Comment:
why not to search the input[type=submit] for all forms and not adding submits. in this way you can test also the form functionality